TwitchPlaysAssembly/Src/TwitchGame.cs

```diff
@@ -23,6 +23,8 @@ public class TwitchGame : MonoBehaviour
 	public readonly Dictionary<int, string> NotesDictionary = new Dictionary<int, string>();
 	public Dictionary<string, Dictionary<string, double>> LastClaimedModule = new Dictionary<string, Dictionary<string, double>>();
 	public readonly List<CommandQueueItem> CommandQueue = new List<CommandQueueItem>();
+	public int callsNeeded = 1;
+	public int callsTotal = 0;
```

Comment:
These should get reset in OnEnable so that they don't carry over to the next round.